catalog/core/catalog-core-solr/src/test/java/ddf/catalog/source/solr/SolrProviderRealTimeQueryTest.java

```diff
@@ -76,7 +75,7 @@ public static void beforeClass() throws Exception {
     // Set to false for real time query by ID tests
     store.setForceAutoCommit(false);
     String solrDataPath = Paths.get("target/surefire/solr/realtime").toString();
-    System.getProperty("solr.data.dir", solrDataPath);
+    System.setProperty("solr.data.dir", solrDataPath);
```

Comment:
❓ Should we use the `RestoreSystemProperties` JUnit rule to properly reset that property?

Reply:
❗️ System properties do propagate in the Java VM, therefore to avoid bleeding them to the next set of tests that might be running inside the same VM, we should make sure to reset them.

Reply:
You can see other Solr tests that set system properties also unset them for the reason he mentioned.